src/fetch/payouts.py

```diff
@@ -410,6 +417,16 @@ def construct_payout_dataframe(
         merged_df["eth_slippage_wei"].fillna(0) + merged_df["network_fee_eth"]
     )
 
+    # 5. Compute buffer accounting target
+    merged_df["buffer_accounting_target"] = np.where(
+        merged_df["pool_address"] == COW_BONDING_POOL.address,
```

Comment:
Should this be `!=` ?

Reply:
Oh, true. I had read this as gnosis bonding pool and treated that as special case.

So the general approach is to use the solver address and the COW DAO is treated as an exception?